api/controllers/AuthController.js

```diff
@@ -45,6 +46,9 @@ module.exports = {
         return res.redirect('/');
       });
     })(req, res);
+  },
 
+  modAuth: function(req, res) {
+    res.redirect("https://www.reddit.com/api/v1/authorize?client_id=" + sails.config.reddit.clientID + "&response_type=code&state=" + req.session.state + "&redirect_uri=" + encodeURIComponent(sails.config.reddit.redirectURL) + "&duration=permanent&scope=flair,modflair,modcontributors,wikiread,wikiedit,read,modposts");
   }
 };
```

Comment:
This will callback to a page with an error message, which I couldn't figure out how to fix easily. The authentication still works, though.